### PR TITLE
Refactor/147 reflect api changes of asset service

### PR DESCRIPTION
**Pull request by storkandstars**

### CODE REVIEW

File: lib/data/model/network_wallet.dart [hunk 1]
```diff
@@ -10,6 +10,7 @@ class NetworkWallet extends Equatable {
   final String address;
   final String? contractAddress;
   final BigInt balance;
```
Comment (jvrastil) [inline lib/data/model/network_wallet.dart:12]: by the way, didn't we discussed this with @PitTxid that we will use string for sending balances?

**Review by jvrastil — APPROVED** [review]

File: lib/data/model/network_wallet.dart (hunk 1 above)
Comment (storkandstars) [reply lib/data/model/network_wallet.dart:12]: Yes. Network types will be strings.

File: lib/data/model/network_wallet.dart [hunk 2]
```diff
@@ -26,16 +28,21 @@ class NetworkWallet extends Equatable {
...
           uuid: const Uuid().v4(),
           networkId: networkId,
           address: address,
           contractAddress: contractAddress,
           balance: balance ?? BigInt.zero,
+          decimalPlaces: decimalPlaces,
           isEnabled: isEnabled,
         );
 
+  bool get isToken => contractAddress != null || contractAddress!.isNotEmpty;
```
Comment (TomasPetrik) [inline lib/data/model/network_wallet.dart:43]: Would be simplier to use contractAddress?.isNotEmpty

Comment (storkandstars) [reply lib/data/model/network_wallet.dart:43]: It works the same, but your suggestion is clearer. I will refactor it in task #141

Comment (TomasPetrik) [reply lib/data/model/network_wallet.dart:43]: Okay

**Pull request merged**